src/PowerShellEditorServices/PowerShellEditorServices.csproj

```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <AssemblyTitle>PowerShell Editor Services</AssemblyTitle>
     <Description>Provides common PowerShell editor capabilities as a .NET library.</Description>
-    <TargetFrameworks>netstandard1.6;net451</TargetFrameworks>
+    <TargetFrameworks>netstandard1.6;net452</TargetFrameworks>
```

Comment:
I wonder if it would be worth it to move to `netstandard2.0` for .NET Core builds?

Reply:
I have been wondering the same. Would there be any reason not to?

Reply:
What version of .NET Core did 6.0 & 6.1 ship with?  Seems like at least 6.0 was 1.x so maybe we're stuck with 1.6 until we're willing to drop support for 6.0 (and maybe 6.1).  Doh!

Reply:
PSCore 6.0 and 6.1 both run on .Net Core 2.1 and implement netstandard2.0.
But I have been told not to submit a PR because the `2.0.0` branch upgrades it to .netstandard2.0 and PSStandard already. PR #830 got closed for that reason for example.

Reply:
They're both compatible I'm pretty sure. The real work in the upgrade was eliminating net45 stuff and things PSStandard didn't like. In which case I think we may as well upgrade to netstandard2.0. But @TylerLeonhardt may have good reasons not to

Reply:
6.0 ships with .NET Core 2.0

Reply:
Unless something changed in the last patch version